packages/react-components/src/components/MessageThread.tsx

```diff
@@ -60,7 +60,7 @@ const isMessageSame = (first: ChatMessage, second: ChatMessage): boolean => {
     JSON.stringify(first.createdOn) === JSON.stringify(second.createdOn) &&
     first.senderId === second.senderId &&
     first.senderDisplayName === second.senderDisplayName &&
-    first.status === second.status
+    JSON.stringify(first.editedOn) === JSON.stringify(second.editedOn)
```

Comment:
It would be the same as it was before as we had the content check already. I've created a user story to investigate it more as we probably need to update morefunctionality here.